### DISCUSSION TITLE
Help...

### DISCUSSION THREAD

**Discussion by MortThePersian** [XTLS/Xray-core#1955] [Q&A]
Dear RPRX;

Restrictions over the Internet in Iran have become even more tight & more complex than before.

I would like to ask, is it possible to speed up the progress of Xray-core v1.8 update that includes "Reality" ?!

Please consider a closer look at the Iranian situation, Since as far as We aware of, GFW is the main tool used to confine communications. What's happening in here now could be a clue to what you will face in the future...

If you ever decided to search for information about Iran, Please be wide-awake !

Sincerely

Behold, I send you forth as sheep in the midst of wolves: be ye therefore wise as serpents, and harmless as doves.
Matthew 10:16

**Comment by RPRX** [comment]
We will release Xray-core v1.8.1 as soon as possible.

**Reply by MortThePersian** [reply]
Godspeed...

**Reply by FranzKafkaYu** [reply]
> We will release Xray-core v1.8.1 as soon as possible.

good job,thanks RPRX~

**Comment by computerscot** [comment]
Some users in Iran are already using REALITY. See "How to make XTLS (V2ray) server for Iran on Ubuntu/Debian with the new Reality protocol" https://github.com/net4people/bbs/issues/243

**Reply by MortThePersian** [reply]
Thank you ! this is exactly what many are looking for...

**Comment by nursery01** [comment]
Can you describe the general working mechanism of Iran's firewall?
<br>

Edited

ok，I understood now.

This situation is very difficult.

**Reply by MortThePersian** [reply]
Hello there, I tried to briefly explain the situation in here;

https://github.com/XTLS/Xray-core/discussions/1961
